tests/lib/Controller/LogpeekTest.php

```diff
@@ -133,8 +132,9 @@ public function testMainWithTag(): void
             ['tag' => $this->session->getTrackID()]
         );
 
-        $c = new Controller\Logpeek($this->config, $this->session);
+        $c = new Controller\Logpeek($this->config, $this->session, $this->moduleConfig);
```

Comment:
This is actually 'wrong';
We either have to pass the config through the constructor, or use the setter two lines below.
The setter is probably the way to go for now, until our module-system allows us to dynamically register services.. The config could then be passed as a service by the routing system, the same way we pass the configuration/session classes